src/main/menu.js

```diff
@@ -204,6 +215,28 @@ export function setupMenu(store) {
                 store.dispatch(setSetting('condencedSidebar', item.checked));
               }
             },
+            { type: 'separator' },
+            // Language menu point
+            {
+              label: i18n.formatMessage({
+                id: 'language'
+              }),
+              submenu: i18n.getConfig('availableLanguages').map(lang => ({
+                label: lang.name,
```

Comment:
Is this showing lowercase languages in the menu? Perhaps we could install `ucfirst` to transform the first char to uppercase? What do you think @sallar?

Reply:
Nevermind, I saw that they're capitalised in a later file.